rules/cloud/aws/aws_ec2_vm_export_failure.yml

```diff
@@ -16,10 +17,7 @@ detection:
     errorMessage: '*'
   filter2:
     errorCode: '*'
-  filter3:
```

Comment:
I don't know about getting rid of filter 3, I would get rid of filter 1 and filter 2, because they are kind of redundant and unnecessary, and all of the SIEM tools I've tested this on, you are able to view the error codes. But I will tag @frack113 on this.

Reply:
If you have allways a errorMessage and errorCode  in the same time in log you need only one.
but if a success got a errorCode:0 it is not a attempt 😄